Documentation/README.md

```diff
@@ -36,7 +36,7 @@
 - Developer Documentation
   - [API usage](./sources/api-usage.md#Message Feed and Code-based experiences APIs)
```

Comment:
Formatting needs fixing here and we can remove `Feed` and state Content Card

Screenshot : 
<img width="821" alt="Screenshot 2024-08-19 at 8 01 47 AM" src="https://github.com/user-attachments/assets/90862b9b-d5f7-44bb-ae9f-0d01e11b7e00">

Reply:
good catch - there are 3 on this page that are broken - fixed them all